sw/airborne/firmwares/rotorcraft/guidance/guidance_indi_hybrid.c

```diff
@@ -71,6 +80,14 @@ struct guidance_indi_hybrid_params gih_params = {
   .heading_bank_gain = GUIDANCE_INDI_HEADING_BANK_GAIN,
 };
 
+float scheduled_pos_gain = 0.;
+float scheduled_pos_gainz= 0.;
+float scheduled_speed_gain= 0.;
+float scheduled_speed_gainz= 0.;
+float landing_slope= 15.0;
```

Comment:
Change to 30 deg

Reply:
30 deg means 0.6m altitude drop every horizontal meter. say the rope is 2 meter in alt and say the drone is 2.5 m away in the horizontal. The drone is going to try to intercept the GS at 3.4, and it is going to overshoot slightly as we observed. Isn't that too high?